src/DatePicker/index.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import generatePicker from 'antd/es/date-picker/generatePicker'
+import en from 'antd/es/date-picker/locale/en_US'
+import { CaretDown } from 'phosphor-react'
+import dateFnsGenerateConfig from 'rc-picker/lib/generate/dateFns'
+
+import './style.css'
+en.lang.quarterFormat = "'Q'Q"
+
+const DatePicker = generatePicker<Date>(dateFnsGenerateConfig)
+
+export const QuarterPicker = DatePicker.QuarterPicker
+
+DatePicker.defaultProps = {
+  suffixIcon: <CaretDown color="#737373" />
+}
+
```

Comment:
Can add defaultProps for quarter picker as well. There you can add:
`format: "YYYY-'Q'Q"`